web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -11,6 +11,12 @@
  * @ingroup farm_nfa
  */
 class ForestPlanGfwForm extends FormBase {
+  
+  protected $routeMatch;
```

Comment:
Please check how other class variables are declared thorough the Drupal standards. Your IDE should be warning you about this, you can configure VSCode or PHPStorm with https://www.drupal.org/docs/develop/standards, see https://www.drupal.org/project/coder